### ISSUE TITLE
[translations] marking English not to translate

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-website#255]
We say to translate the words but not the acronym
 https://github.com/w3c/translation-glossaries/blob/main/general.md
e.g., Richtlinien für barrierefreie Webinhalte (WCAG)

afaik, Acronyms and other things that should not be translated should be marked up:
* lang="en"
* translate="no" (see https://www.w3.org/International/questions/qa-translate-flag.en)

To Do:
* [ ] research best way to do this
* [ ] add to translation instructions
* [ ] update all English pages that have such
* [ ] update all translations with it
phew!

**Comment by r12a** [comment]
There's a slight ambiguity in the text above. If "Acronyms and other things that should not be translated should be marked up" should be read as "Acronyms that should not be translated and other things that should not be translated should be marked up" then ok.  Just wanted to remind that some acronyms do need to be translated, eg. NATO -> OTAN in Spanish.

**Comment by shawna-slh** [comment]
updated to:
> "afaik, Things that should not be translated should be marked up:
